### PR TITLE
Update svls to v0.1.16

### PR DESCRIPTION
**Pull request by dalance**
This PR updates svls to v0.1.16.

### CODE REVIEW

**Comment by dalance** [comment]
This PR depends on #272.

File: installer/install-svls.cmd [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
-del svls-%VERSION%-x86_64-win.zip
+@echo off
+
+setlocal
+set VERSION=v0.1.16
+curl -L -o svls-%VERSION%-x86_64-win.zip "https://github.com/dalance/svls/releases/download/%VERSION%/svls-%VERSION%-x86_64-win.zip"
+call "%~dp0\run_unzip.cmd" svls-%VERSION%-x86_64-win.zip
+move target\x86_64-pc-windows-msvc\release\svls.exe .
+rd /Q /S target
+del svls-%VERSION%-x86_64-win.zip
```
Comment (mattn) [inline installer/install-svls.cmd:9]: Please keep fileformat being DOS format.

Comment (dalance) [reply installer/install-svls.cmd]: Sorry. Could you check again?

**Comment by mattn** [comment]
Thank you!

**Pull request merged**